requirements.txt

```diff
@@ -1,2 +1,2 @@
 dice<=1.1.0
-pillow
\ No newline at end of file
+pillow==4.1.1
```

Comment:
Freezing the pillow version might clash with other beards, can we use `pillow>=4.1.1`?